src/presentation/http/router/noteList.test.ts

```diff
@@ -1,27 +1,40 @@
-import userSessions from '@tests/test-data/user-sessions.json';
+import type User from '@domain/entities/user';
 import { describe, test, expect, beforeAll } from 'vitest';
 
-
-/**
- * Access token that will be used for Auhorization header
- */
 let accessToken = '';
+let user: User;
 
-describe('Note list API', () => {
-  beforeAll(() => {
+describe('NoteList API', () => {
+  beforeAll(async () => {
```

Comment:
I guess we need to truncate table and do other stuff before each test to make them independent